src/components/silhouette-pane.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useMemo, useState } from "react";
+import { clsx } from "clsx";
+
+import { PaneTitle } from "./pane-title";
+import { AppContext } from "../hooks/use-app-context";
+import { ISilhouetteData, ISilhouetteOrganData, silhouetteData, silhouetteOrganData }
+  from "../assets/app-data/silhouette-data";
+
+import "./silhouette-pane.scss";
+
+// Make sure you update $transition-duration in silhouette-pane.scss if you change this.
+const zoomDuration = 2000;
+
+interface ISilhouettePane {
+  ac: AppContext;
+  hasZoomed: boolean;
+  setHasZoomed: (val: boolean) => void;
+}
+export const SilhouettePane = ({ ac, hasZoomed, setHasZoomed }: ISilhouettePane) => {
+  const [silhouetteStyle, setSilhouetteStyle] = useState<Record<string, any>>({});
+  const [buttonStyle, setButtonStyle] = useState<Record<string, any>>({});
+  const [organStyle, setOrganStyle] = useState<Record<string, any>>({});
+  const [zooming, setZooming] = useState(false);
+
+  // Determine which silhouette to use
+  const sdIndex = useMemo(() => {
+    const index = Math.floor(Math.random() * silhouetteData.length);
+    return index < silhouetteData.length ? index : 0;
```

Comment:
```suggestion
    return Math.floor(Math.random() * silhouetteData.length);
```
Shouldn't need the additional test as `Math.random()` returns value in [0,1].